application/models/Files.php

```diff
@@ -216,4 +235,24 @@ public function getComments()
 							   ->exec();
 			return $results;
 		}
+		
+		public static function search($string)
+		{
+			if(empty($string))
+				return false;
+			
+			$searchArray = explode(' ', htmlspecialchars($string));
+			
+			$like = '%' . current($searchArray) . '';
+			$in = current($searchArray);
+			array_shift($searchArray);
```

Comment:
Cette méthode est très floue sur ce fonctionnement : je ne comprends pas vraiment sa logique ^^'

Pourquoi le premier élément de `$searchArray` est stockée dans `$like` alors que le deuxième est dans `$in` ? Pourquoi ensuite supprimer le premier élément de `$searchArray` ?